math_errors.py

```diff
@@ -9,40 +9,40 @@ class TestMathErrors(unittest.TestCase):
     """
     
     def test_00(self):
-        x = 1 + 1
+        x = 1 + 0
         answer = 1
         self.assertEqual(x, answer)
 
     def test_01(self):
-        x = 100 / 2
+        x = 100 * 2
         answer = 200
         self.assertEqual(x, answer)
 
     def test_02(self):
         x = 0
-        for i in range(10):
+        for i in range(4):
```

Comment:
`range(4)` evaluates to [0, 1, 2, 3]. The value of `answer` would need to match the sum of those numbers for this test to pass.